terraform/aws/modules/customer-web-server/web-server.tf

```diff
@@ -313,6 +313,54 @@ resource "kubernetes_ingress" "customer_web_server" {
         }
       }
     }
+    tls {
+      hosts = [var.cws_ingress]
+    }
+  }
+
+  lifecycle {
+    ignore_changes = [
+      metadata,
+      spec,
```

Comment:
If I'm not mistaken we're not including check for metadata and spec in this case because we could change those in other parts or with other workflows and we don't want to be reset back with tf right?

Reply:
Yeap exactly :)